pkg/storage/utils/decomposedfs/spaces.go

```diff
@@ -434,14 +434,13 @@ func (fs *Decomposedfs) ListStorageSpaces(ctx context.Context, filter []*provide
 
 // MustCheckNodePermissions checks if permission checks are needed to be performed when user requests spaces
 func (fs *Decomposedfs) MustCheckNodePermissions(ctx context.Context, requestedUserID string, unrestricted bool) bool {
```

Comment:
Thanks for updating the comment. It is now clear to me. But that makes `requestedUserID` obsolete, as it isn't used any more, right?

Reply:
yes, let me cleanup.